Add DIST output to the KDIST and NDIST

RomanIlchenko1308 opened the pull request:
Improvement of the `mapdl.kdist` and `mapdl.ndist` with the implementation of the DIST output. 
To obtain the distance between keypoints and nodes without additional hand calculations.

akaszynski commented on ansys/mapdl/core/_commands/parse.py:
```diff
@@ -99,3 +99,9 @@ def parse_output_volume_area(msg):
+    """Parse the node value from a node message"""
+    finds = re.findall(NUM_PATTERN, msg)[-4:]
+    if len(finds) == 4:
+        return [float(val) for val in finds]
```
Should have an additional blank line at the end of the file. See:
https://www.flake8rules.com/rules/W391.html

RomanIlchenko1308 replied:
Thank you, Alex(@akaszynski),
I have changed.

RomanIlchenko1308 commented:
Hi @germa89 may I ask you to check this PR where I have added full output functionality with write description for NDIST and KDIST. Of course for new features I have added unit testing too.

germa89 reviewed (approved):
Good job!

germa89 commented:
Approved. @RomanIlchenko1308  feel free to hit the green ``Squash and merge`` button.
